commit 544a175f04f32a87d7a48c9540fca67ce84c253f
Author: John Johansen <john.johansen@canonical.com>
Date:   Sun Sep 3 12:10:20 2017 -0700

    UBUNTU: SAUCE: LSM stacking: fixup: alloc_task_ctx is dead code
    
    Fixes: 7b27ec622c90 ("LSM: manage credential security blobs)
    Signed-off-by: John Johansen <john.johansen@canonical.com>
    Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/security/apparmor/context.c b/security/apparmor/context.c
index db203ee24db8..8956da14b462 100644
--- a/security/apparmor/context.c
+++ b/security/apparmor/context.c
@@ -29,16 +29,6 @@
 #include "include/context.h"
 #include "include/policy.h"
 
-/**
- * aa_alloc_task_context - allocate a new task_ctx
- * @flags: gfp flags for allocation
- *
- * Returns: allocated buffer or NULL on failure
- */
-struct aa_task_ctx *aa_alloc_task_context(gfp_t flags)
-{
-	return kzalloc(sizeof(struct aa_task_ctx), flags);
-}
 
 /**
  * aa_free_task_context - free a task_ctx
diff --git a/security/apparmor/include/context.h b/security/apparmor/include/context.h
index c6d5dbbd18b0..349ea3088267 100644
--- a/security/apparmor/include/context.h
+++ b/security/apparmor/include/context.h
@@ -45,7 +45,6 @@ struct aa_task_ctx {
 	u64 token;
 };
 
-struct aa_task_ctx *aa_alloc_task_context(gfp_t flags);
 void aa_free_task_context(struct aa_task_ctx *ctx);
 void aa_dup_task_context(struct aa_task_ctx *new,
 			 const struct aa_task_ctx *old);
